go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/jasonjoo2010/goschedule
 go 1.14
 
 require (
-	github.com/google/uuid v1.1.1
+	github.com/google/uuid v1.1.2
 	github.com/robfig/cron/v3 v3.0.0
 	github.com/sirupsen/logrus v1.6.0
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
-	golang.org/x/sys v0.0.0-20210423082822-04245dca01da // indirect
+	go.etcd.io/etcd/client/v3 v3.5.0 // indirect
```

Comment:
Because goschedule supports multiple storages we cannot introduce the dependency directly into the main repository. We will introduce and upgrade the corresponding store implementations instead.

Reply:
will do